lib/jsonapi_plug/normalizer.ex

```diff
@@ -215,6 +210,33 @@ defmodule JSONAPIPlug.Normalizer do
     end)
   end
 
+  defp find_related_relationships(
+         %Document{} = document,
+         list,
+         resource,
+         conn
+       ) do
+    Enum.reduce(
+      list,
+      [],
+      fn %ResourceIdentifierObject{id: id} = data, acc ->
+        case find_related_relationship(document, data, resource, conn) do
+          nil ->
+            [
+              %{
+                id: id
+              },
+              acc
+            ]
+
+          item ->
+            [Map.put(item, "id", id) | acc]
```

Comment:
This looks like an error. One side is creating a list of lists the other side is concatenating to a list.